src/components/rockets/Rockets.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRockets } from '../../redux/rockets/rocketsSlice';
+import RocketLi from './RocketLi';
+import '../../styles/rockets/rocket.css';
+
+export default function Rockets() {
+  const { rockets, isRocketLoading } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!rockets.length) {
+      dispatch(getRockets());
+    }
```

Comment:
As mentioned in your task, data should be fetched at the start of the webapp:
![fetchWhenStarts](https://user-images.githubusercontent.com/114868555/230939475-5e977d16-1ecf-4ec9-bd58-565c67349c59.JPG)
- Consider adding `dispatch(getRockets)` at `/index.js`

Reply:
Our `dispatch(getRockets)` are link as [index](https://reactrouter.com/en/main/route/route#index) in routes so its get called first automatically whenever we render index.js or App.js.
![Screenshot from 2023-04-10 21-50-11](https://user-images.githubusercontent.com/70534992/230945468-13f50d3e-5365-4edb-bfd3-84a9ccfcf291.png)